utils/utils.js

```diff
@@ -1,9 +1,9 @@
 export const colors = [
-  "yellow",
-  "green",
-  "lightblue",
-  "hotpink",
-  "red",
+  "#FF009A", // hotpink
+  "#009E10", //grün
+  "#385CFF", //blau
+  "#B55A10", //braun
+  "#FF110A", //rot
```

Comment:
I am not sure, where you use this. But if it is meant for styling theming, then it is better practice to use these variables as css variables (var(--color1)).

And then put the colors to the Globalstyles:
```css
:root {
   -color1: "#FF009A";
   -color2: "#009E10";
   -color3: "#385CFF";
   -color4: "#B55A10";
   -color5: "#FF110A";
}
```

Reply:
These are the colors for my ColorCode(maybe I should call it better).

These colors are only for the ColorCode and are not used otherwise. I'm not quite sure if it is useful to outsource them, but maybe I have to do it anyway with my dark mode😅.